app/Resources/views/base.html.twig

```diff
@@ -2,10 +2,10 @@
 <html>
     <head>
         <meta charset="UTF-8" />
-        <title>{% block title %}Welcome!{% endblock %}</title>
+        <title>{% block title %}Prezent Foosball competition{% endblock %}</title>
```

Comment:
I want to keep this as generic as possible, so remove Prezent here. Will add variable to parameter.yml for site name later on